sys-auth/pam_u2f/pam_u2f-1.4.0.ebuild

```diff
@@ -15,11 +15,12 @@ KEYWORDS="~amd64 ~arm64 ~x86"
 IUSE="test"
 RESTRICT="!test? ( test )"
 
-DEPEND="
+BDEPEND="app-text/asciidoc"
+RDEPEND="
```

Comment:
This makes changes that aren't reflected in the commit message.

Reply:
(The fact that those changes are also wrong, is independent of the fact that they aren't reflected in the commit message. Documenting the purpose of the change would likely result in coming up with reasons to not make the change.)

Reply:
Thanks. Some confusion with the `?DEPEND` variables on my side.